edx_solutions_api_integration/users/views.py

```diff
@@ -2,7 +2,11 @@
 
 import json
 import logging
+
+from course_metadata.models import CourseAggregatedMetaData
```

Comment:
Import order here does not comply with [edx strandards](http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python_guidelines.html#imports-order).